patches/server/0750-Improve-EntityTransformEvent-cancellation-handling.patch

```diff
@@ -0,0 +1,41 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jake Potrebic <jake.m.potrebic@gmail.com>
+Date: Sat, 21 Aug 2021 21:15:38 -0700
+Subject: [PATCH] Improve EntityTransformEvent cancellation handling
+
+
+diff --git a/src/main/java/net/minecraft/world/entity/monster/Slime.java b/src/main/java/net/minecraft/world/entity/monster/Slime.java
+index e1a593b464c35f68b22e84a09f99ee72af73da32..8a1365a79aa4121e47fb257ed2066a8cfd541cba 100644
+--- a/src/main/java/net/minecraft/world/entity/monster/Slime.java
++++ b/src/main/java/net/minecraft/world/entity/monster/Slime.java
+@@ -252,6 +252,7 @@ public class Slime extends Mob implements Enemy {
+ 
+             // CraftBukkit start
+             if (CraftEventFactory.callEntityTransformEvent(this, slimes, EntityTransformEvent.TransformReason.SPLIT).isCancelled()) {
++                super.remove(reason); // Paper - remove entity even when transform is cancelled (fixes SPIGOT-6713)
```

Comment:
Well craftbukkit already fix this in same time for this PR?
https://hub.spigotmc.org/stash/projects/SPIGOT/repos/craftbukkit/commits/c949675e1fc50557752110f12e2281c1105da667

Reply:
Yep, that will get removed. But the other fix that they totally did not fix is still needed.